content/2019-05-05_Sprint-na-vychode.md

```diff
@@ -0,0 +1,25 @@
+Title: Sprint na východě
+Date: 2019-05-05 00:00:00
+Author: Karolina Surma
+
+## Sprint? To jako běháte?
+
+Členové a členky české pythonní komunity se obvykle dvakrát do roka sejdou v nějakém koutu republiky. Letos jsem se podruhé vydala na společné řešení komunitních otázek, webových aplikací a ostatních aktivit kolem propagace programování v Pythonu. Jak je známo, začátečníci mají obvykle řadu obav před zapojením se mezi zkušené programátory a programátorky. Jak je taky známo, Pythonní komunita je inkluzivní a pořádá řadu aktivit mířených přímo na nováčky. Mírou úspěchu může být tedy nadpoloviční účast brněnských PyLadies (v hojném počtu 4) vedle zkušenějších kolegů z Moravy a Slezska. 
+Jarní, komornější 7-členná akce se konala v Lískovci u Frýdku-Místku. Aktivnější sprinteři mohli zahájit maraton již během týdně návštěvou ostravského kurzu PyLadies, a následného Pyva. K víkendu jsme se v konečném počtu potkali přímo u Lumíra, který nám zapůjčil své prostory pro intenzivní kódění a brainstorming. 
+
```

Comment:
V delších textech se mi moc nelíbí použití 7-členná, napsala bych sedmičlenná, ale to je čistě o osobní preferenci :)

"již během týdně" - " již během týdne"
"ostravského kurzu PyLadies, a následného Pyva" - "ostravského kurzu PyLadies a následného Pyva"